webapp/src/app.tsx

```diff
@@ -4979,11 +4980,11 @@ export class ProjectView
         if (this.isTutorial()) {
             pxt.tickEvent("tutorial.editorLoaded")
             this.postTutorialLoaded();
+        }
 
-            pxt.perf.recordMilestone("editorContentLoaded");
-            if (!this.autoRunOnStart()) {
-                pxt.analytics.trackPerformanceReport();
-            }
+        pxt.perf.recordMilestone(Milestones.EditorContentLoaded);
+        if (!this.autoRunOnStart()) {
```

Comment:
Why is this no longer nested?

Reply:
I wanted to send this milestone to minecraft regardless of whether the project being loaded is a tutorial.